docs/java-api-quickstart.md

```diff
@@ -38,37 +38,42 @@ The Hive catalog connects to a Hive metastore to keep track of Iceberg tables.
 You can initialize a Hive catalog with a name and some properties.
 (see: [Catalog properties](../configuration/#catalog-properties))
 
-**Note:** Currently, `setConf` is always required for hive catalogs, but this will change in the future.
+### Using a Hive catalog
```

Comment:
Looks like the changes are incorrect here. This now duplicates the previous section heading and paragraph.